django_mysql/forms.py

```diff
@@ -240,8 +240,6 @@ class JSONField(forms.CharField):
     widget = forms.Textarea
 
     def to_python(self, value):
-        if self.disabled:
```

Comment:
removing this logic isn't the right approach, we want to work with disabled on Django 1.9+. instead we should do:

```
if getattr(self, 'disabled', False):
```

or

```
if django.VERSION[:2] >= (1, 9) and self.disabled:
```

Allowing the code to work on both versions. And add a test too, to get coverage of the lines.